pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: ../common/yamllint
+platforms:
+  - name: instance
+    image: solita/ubuntu-systemd:latest
+    command: /sbin/init
+    capabilities:
+      - SYS_ADMIN
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    network_mode: ${NETWORK_MODE:-bridge}
```

Comment:
Is custom `network_mode` really needed here?

Reply:
Yes, necessary in some circumstances (at least in my dev environment). Will think about that more.